app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Post;
+use App\Models\User;
+use Auth;
+use Illuminate\Http\Request;
+use App\Http\Requests\PostRequest;
+
+class PostController extends Controller
+{
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $users = User::withCount('posts')->get();
+
+        return view('home', compact('users'));
+    }
+
+    public function show($id)
+    {
+        $user = User::findOrFail($id)->load('posts');
+
+        return view('posts.show', compact('user'));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return view('posts.create');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(PostRequest $request)
+    {
+        $post = new Post;
+        $name = $request->name;
+        $user_id = Auth::id();
+        $result = $post->create([
+            'name' => $name,
+            'user_id' => $user_id
```

Comment:
```suggestion
            'user_id' => Auth::id();
```
cái này dùng 1 lần thì ko cần tạo biến nhé

Reply:
Dạ vâng ạ

Reply:
Những biến dùng 1 lần thì em fix không tạo biến nữa mà dùng thẳng luôn rồi ạ